data/interfaces/default/js/tables/history_table.js

```diff
@@ -98,14 +98,15 @@ history_table_options = {
             "createdCell": function (td, cellData, rowData, row, col) {
                 if (cellData !== '') {
                     var transcode_dec = '';
-                    if (rowData['video_decision'] === 'transcode') {
+                    if (rowData['video_decision'] === 'transcode' || rowData['audio_decision'] === 'transcode') {
```

Comment:
`rowData['audio_decision']` doesn't exist here yet. Needs to be sent with `get_history`